### DISCUSSION TITLE
Add assets in Visual Editor

### DISCUSSION THREAD

**Discussion by fraveman** [requarks/wiki#5025] [Help / Questions] [answered]
Hi guys,
anyone knows how to add a file/asset to the visual editor? There is a corrisponding button in the code editor, but i cannot find anything in the visual editor. 
Thanks

Specs:
Wiki.js 2.5.275
Node.js 16.13.2
PostgreSQL 11.15
OS: Docker Container (Linux) on Ubuntu 20.04.3

**Comment by NGPixel** [comment, marked as answer]
<img width="372" alt="image" src="https://user-images.githubusercontent.com/15522395/154825663-ae2baa09-6af4-478c-a77f-3d447a18ee25.png">

**Reply by fraveman** [reply]
🤦‍♂️🤦‍♂️🤦‍♂️
thanks
